<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Panel</class>
 <widget class="QMainWindow" name="Panel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>551</width>
    <height>322</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Panel</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="RadioButtons">
        <item>
         <widget class="QRadioButton" name="radiobutton_point1">
          <property name="text">
           <string>点1(左上)</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radiobutton_point2">
          <property name="text">
           <string>点2(右上)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radiobutton_point3">
          <property name="text">
           <string>点3(右下)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radiobutton_point4">
          <property name="text">
           <string>点4(左下)</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="Sliders">
        <item>
         <layout class="QHBoxLayout" name="SliderX">
          <item>
           <widget class="QLabel" name="label_x">
            <property name="text">
             <string>X坐标(宽度方向)</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="slider_x">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="SliderY">
          <item>
           <widget class="QLabel" name="label_y">
            <property name="text">
             <string>Y坐标(高度方向)</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="slider_y">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="PushButtons">
        <item>
         <widget class="QPushButton" name="pushbutton_save">
          <property name="text">
           <string>保存</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
